### PR TITLE
Feature/86eq301wd/multiple contents approval rejections

### PR DESCRIPTION
**Pull request by acostales08**
initial push

### CODE REVIEW

File: packages/core-library/system/app/internal/blocks/Hub/content/approval/steps/content/regular/ApprovalListViewBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+import { AuthorizedContentsResponseType } from "../../../../../../../../../../api/types";
+import { useAtom } from "jotai";
+import { actionButtons } from "../../../../../Settings/SettingsManagement/constants/constants";
+import {
+  ContentDateAtom,
+  ContentDateType,
+} from "../../../../../../../../../../components/Dialog/DialogFormBlocks/contentApproval/validation";
+import { useEffect, useState } from "react";
+import { ApprovalListView } from "./ApprovalListView";
+import { mockData } from "./mockData";
```
Comment (DropitupQwerty) [inline packages/core-library/system/app/internal/blocks/Hub/content/approval/steps/content/regular/ApprovalListViewBlock.tsx:30]: are you still using the mock data?

File: packages/core-library/system/app/internal/blocks/Hub/content/approval/steps/content/regular/ApprovalListViewBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+          showToast("No Selected Content", "info");
+        }
+        break;
+      default:
+        console.warn("Unknown action:", action);
+    }
+  };
+
+  const renderActionButtons = (contentId: string, contentAuthorId: string) =>
+    actionButtons.map((btn) => {
```
Comment (DropitupQwerty) [inline packages/core-library/system/app/internal/blocks/Hub/content/approval/steps/content/regular/ApprovalListViewBlock.tsx:126]: Add actionButtons.length > 0

File: packages/core-library/system/app/internal/blocks/Hub/content/approval/steps/content/regular/ApprovalListViewBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  }
+
+  return (
+    <ApprovalListView
+      multiple={multiple}
+      handleMultipleSelection={handleMultipleSelection}
+      selectedRows={selectedRows ?? 0}
+      handleSelection={handleSelection}
+      columns={columns}
+      data={mockData ?? []}
```
Comment (DropitupQwerty) [inline packages/core-library/system/app/internal/blocks/Hub/content/approval/steps/content/regular/ApprovalListViewBlock.tsx:210]: can we use the data from query?

File: packages/core-library/system/app/internal/blocks/Hub/core/hooks/useContentLoader.tsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { usePageLoaderContext } from "../../../../../../../contexts/PageLoaderContext";
+
+export const useContentLoader = () => {
```
Comment (DropitupQwerty) [inline packages/core-library/system/app/internal/blocks/Hub/core/hooks/useContentLoader.tsx:3]: Can i ask why we need to create a another custom hook when we can use the usePageLoaderContext directly?

**Review by DropitupQwerty — APPROVED** [review]

**Review by JezChristian — APPROVED** [review]

**Pull request merged**